setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name="imsms_analysis",
+    entry_points={
+        'q2_mlab.models': [
+            'RandomForestPCA=imsms_analysis.custom_models:RandomForestPCA'
```

Comment:
Should this say RandomForestSVD?

Reply:
Yep - I haven't been able to run analysis.py perfectly yet but switching this to SVD plays nicer with the unadulterated biom tables.